app/upgrades.go

```diff
@@ -200,6 +201,30 @@ var upgrades = map[string]appUpgrade{
 			return vm, nil
 		},
 	},
+	"umber-rc1": { // upgrade for v1.19.0-rc1
+		Added: []string{crisistypes.ModuleName},
+		Handler: func(ctx sdk.Context, app *App, vm module.VersionMap) (module.VersionMap, error) {
+			var err error
+			vm, err = runModuleMigrations(ctx, app, vm)
+			if err != nil {
+				return nil, err
+			}
+
+			return vm, nil
+		},
```

Comment:
We also want to run `removeInactiveValidatorDelegations` during the main upgrade and at least one of the rc upgrades for a color.

```suggestion
		Handler: func(ctx sdk.Context, app *App, vm module.VersionMap) (module.VersionMap, error) {
			var err error
			vm, err = runModuleMigrations(ctx, app, vm)
			if err != nil {
				return nil, err
			}
			
			removeInactiveValidatorDelegations(ctx, app)

			return vm, nil
		},
```